read_motif_v6.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Nov  2 01:28:58 2020
+
+@author: YB Moon (ymoon06)
+"""
+"""
+input:
+    motif = motif sequence. must be string and big character
+    read= = read sequence. must be string and big character
+    assume motif < read
+    min_overlap = min threshold for match
+    full_contain= boolean parameter. If True, only report matches that are
+                  fully contained in the read
+"""
+"""
+output:
+    returns a list of tuple(a,b) where:
+    a is overlap start position of read
+    b is fraction of motif that overlaps
+"""
+
+
+def find_overlaps(motif, read, min_overlap, full_contain=False):
```

Comment:
Please add type hints to indicate the data types of the input parameters as well as the return type: https://docs.python.org/3.8/library/typing.html